cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -54,6 +54,9 @@ spec:
         # (Optional) Rook Agent toleration key. Set this to the key of the taint you want to tolerate
         # - name: AGENT_TOLERATION_KEY
         #   value: "<KeyOfTheTaintToTolerate>"
+        # (Optional) Rook Agent tolerations list. Put here list of taints you want to tolerate in YAML format.
+        # - name: AGENT_TOLERATIONS
+        #   value: "- effect: NoSchedule\n  key: node-role.kubernetes.io/controlplane\n  operator: Exists\n- effect: NoExecute\n  key: node-role.kubernetes.io/etcd\n  operator: Exists\n"
```

Comment:
Hmm, I wonder if we could define a multi-line values for the env vars to make it more readable?

Reply:
Unfortunately, that's not possible. I tested that in the first place :) But I don't remember where was the problem exactly :/

Reply:
The following seemed to work for me in a quick test:
```
        - name: ROOK_VAR_TEST
          value: "this\n
            is a test\n
            of a multi-line string"
```

Adding this logging to the operator:
```
logger.Infof("MULTI-LINE STRING: %s", os.Getenv("ROOK_VAR_TEST"))
```

Resulted in this in the log file:
```
2019-05-31 04:23:22.920439 I | operator: MULTI-LINE STRING: this
 is a test
 of a multi-line string
```

Reply:
I need to check how helm is going to handle that, since I wanted there to specify just plain YAML array there. Or is it fine to have pretty-printed example in the docs, but helm will generate one-liner?